pkg/components/velero/component.go

```diff
@@ -201,12 +201,8 @@ func (c *component) getProvider() (provider, error) {
 		providers = append(providers, c.Restic)
 	}
 
-	if len(providers) > 1 {
-		return nil, fmt.Errorf("more than one provider configured")
-	}
-
-	if len(providers) == 0 {
-		return nil, fmt.Errorf("no providers configured")
+	if len(providers) != 1 {
+		return nil, fmt.Errorf("expected 1 provider, got: %d", len(providers))
```

Comment:
As we change this logic in #1149, perhaps we could just remove this from this PR and deal with it in the #1149 ?

Reply:
@invidian done. Removed this commit from here.

This PR now only updates the the component and openebs velero plugin.

Reply:
I mean, removing the the was actually useful :) but can be done in some other PR too :smile: